bug_update.php

```diff
@@ -65,6 +65,9 @@
 
 $f_bug_id = gpc_get_int( 'bug_id' );
 $t_existing_bug = bug_get( $f_bug_id, true );
+$f_update_type = gpc_get_string( 'action_type', BUG_UPDATE_TYPE_NORMAL );
+
+$t_update_action = $f_update_type == BUG_UPDATE_TYPE_NORMAL;
```

Comment:
I'm not sure we actually need this variable, IMO it makes the code more confusing, and I'd rather just have 

```if( $t_cf_def[$t_cf_require_check] && $f_update_type == BUG_UPDATE_TYPE_NORMAL ) {```